plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyDrawOverride.cpp

```diff
@@ -13,7 +13,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+
+// GL loading library needs to be included before any other OpenGL headers.
+// We also disable clang-format for this block, since otherwise v10.0.0 fails
+// to recognize that "ProxyDrawOverride.h" is the related header.
+// clang-format off
+#include <pxr/pxr.h>
```

Comment:
Is this include order intentional?

Reply:
It is, yeah. Internally, we define `USD_VERSION_NUM=PXR_VERSION`, so we need to include `pxr.h` first so that `USD_VERSION_NUM` resolves to a valid value. I had left a comment to that effect in an earlier PR, but didn't apply the same comment here:
https://github.com/Autodesk/maya-usd/blob/51d22f04f685253fe54e7429de1d0a4be49e3b80/lib/mayaUsd/render/mayaToHydra/utils.cpp#L17

The eventual goal would be that we could drop the maya-usd specific `USD_VERSION_NUM` and just use `PXR_VERSION` everywhere, since that's provided by USD. Now that I'm looking at it again, I think we'd actually be ready to do that now. @pmolodo fixed `PXR_VERSION` to make it a useable value for version checks in https://github.com/PixarAnimationStudios/USD/pull/886, which made it into core USD 19.11, so with that as the current minimum core USD version, it should be safe to switch over.

Reply:
This reminds me https://github.com/Autodesk/maya-usd/issues/645

Reply:
> This reminds me #645

Ah, yup, well referenced. :)